packages/core/event.ts

```diff
@@ -57,8 +57,12 @@ export function getEventName(eventNameObj: EventNameObjectType, forAttendeeView
         if (variable === bookingField) {
           let fieldValue;
           if (eventNameObj.bookingFields) {
-            fieldValue =
-              eventNameObj.bookingFields[bookingField as keyof typeof eventNameObj.bookingFields]?.toString();
+            const field = eventNameObj.bookingFields[bookingField as keyof typeof eventNameObj.bookingFields];
+            if (typeof field === "object" && "value" in field) {
```

Comment:
booking fields could also be of type object { value, label } ( in case of attendee-cancelled-email)